designdocs/passkey_p256.md

```diff
@@ -0,0 +1,556 @@
+# Design Doc for P256 PassKey Support
```

Comment:
Looks like the standard is with a lower case K. `/s/assKey/asskey/g`

Reply:
I will change it to `Passkey`